src/main/java/constants/LottoConstants.java

```diff
@@ -0,0 +1,7 @@
+package constants;
+
+public class LottoConstants {
+    public static final int LOTTO_PRICE = 1000;
+    public static final int LOTTO_NUMBERS_SIZE_THRESHOLD = 6;
+    public static final int LOTTO_NUMBER_MAX_THRESHOLD = 45;
+}
```

Comment:
로또에 사용되는 에러/ 상수들을 따로 관리하고 계시네요.
사용되는 객체가 아닌 다른 객체로 관리하는 이유가 따로 있을까요?
또 어떤 장단점이 있을까요?

Reply:
해당 상수는 AnswerLotto, Lotto, LottoService에서 사용하고 있어서 따로 새로운 클래스에 관리하도록 했어요.
LottoConstants.java 를 작성하기전에 했던 고민이 여러가지 있었는데요.

1. 그냥 각자 클래스에 상수를 따로따로 선언하기.
2. Lotto 클래스에 관련 상수를 다 선언해놓고, 다른 클래스에서 가져다 쓰기.
3. (현재 방법) 상수 관련 클래스를 새로 만들기.

1번은 만약 상수 값이 업데이트 되었을 때, 상수가 선언되어 있는 모든 클래스의 값을 업데이트 해줘야 하기 때문에, 그리고 중복 코드가 발생하기 때문에 실행하지 않았고요.
2번은 단기적으론 편리하겠지만 결국에 Lotto 클래스가 Lotto 비즈니스 모델 이외에도 Lotto 관련 상수를 저장하는 책임을 하나 더 갖게 되는 것이고, 장기적으로 불필요한 결합도를 유발할 것 같아 결국엔 3번을 실행했어요.

Reply:
네~ 잘 고민해주셨네요.
이렇게 상수를 다른 클래스로 관리하게 되면 프로그램 규모가 커졌을 때 어떤 상수인지 파악하기 어려울 수 있어요.
각 상수를 의미 있는 도메인 객체 내부에 포함하는 것이 더 적절할 수 있습니다.
- 도메인 객체와 밀접한 관련이 있는 상수라면 해당 객체 내부에 포함하는 것이 좋음. 
	- Money 하위에 LOTTO_PRICE, (이렇게 되면 Money보다는 LottoMoney라고 이름짓는게 더 나아보이네요.) 	
	- Lottos 하위에 LOTTO_NUMBERS_SIZE_THRESHOLD
	- Lotto 하위에 LOTTO_NUMBER_MAX_THRESHOLD
- 여러 곳에서 공통으로 사용하는 상수라면, 유지보수성을 고려해 상수 클래스로 관리하는 것도 좋음.
- 프로그램의 규모에 따라 어느 방식이 더 적절할지 고민하는 것이 중요.

이번 미션은 히스타 취향대로 선택하시면 되겠네용.
```suggestion
public class LottoConstants {
    public static final int LOTTO_PRICE = 1000;
    public static final int LOTTO_NUMBERS_SIZE_THRESHOLD = 6;
    public static final int LOTTO_NUMBER_MAX_THRESHOLD = 45;
    
    private LottoConstants() {
    }
}
```
다만 이렇게 상수를 관리하는 클래스를 사용하실 때는 아래처럼 불필요한 객체 생성을 막아주면 좋습니다.

Reply:
정말 좋은 피드백 감사드립니다! 다른 클래스가 책임을 하나 더 갖게 되는것이라고 해석될 수도 있겠지만, 상수 관리의 편리성을 위해서는 도메인 객체에서 상수를 관리하는 방법이 좋을 것 같네요!